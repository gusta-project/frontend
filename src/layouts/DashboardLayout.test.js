import { render } from '@testing-library/react';
import React from 'react';

import Layout from './DashboardLayout';

describe('Dashboad <Layout />', () => {
  const layoutOptions = {
    border: false,
    header: true,
    title: false,
    style: {}
  };

  it('renders correctly', () => {
    const { asFragment } = render(
      <Layout
        pageTitle="Not Found"
        header="Not Found :("
        options={layoutOptions}
      >
        Children go here
      </Layout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
