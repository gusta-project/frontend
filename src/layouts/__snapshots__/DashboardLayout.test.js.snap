// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboad <Layout /> renders correctly 1`] = `
<DocumentFragment>
  <div
    class="card"
  >
    <div
      class="card-header"
    >
      Not Found :(
    </div>
    <div
      class="card-body"
    >
      Children go here
    </div>
  </div>
</DocumentFragment>
`;
